chat-commands.js

```diff
@@ -399,11 +401,10 @@ exports.commands = {
 
 		user.avatar = avatar;
 		if (!parts[1]) {
-			this.sendReply("Avatar changed to:\n" +
-				'|raw|<img src="//play.pokemonshowdown.com/sprites/trainers/' + (typeof avatar === 'string' ? avatar.substr(1) : avatar) + '.png" alt="" width="80" height="80" />');
+			this.sendReplyBox(`Avatar changed to:\n<img src="//play.pokemonshowdown.com/sprites/trainers/${(typeof avatar === 'string' ? avatar.substr(1) : avatar)}.png" alt="" width="80" height="80" />`);
 		}
 	},
-	avatarhelp: [`/avatar [avatar number 1 to 293] - Change your trainer sprite.`],
+	avatarhelp: [`/avatar [avatar number 1 to 293] - Changes your trainer sprite.`],
```

Comment:
I'm not sure why this was changed, it was grammatically correct beforehand and matches the tense of the other help commands.